batch-index-files/README.md

```diff
@@ -0,0 +1,27 @@
+# AWS batch support for bgzip and the bio-index
+The core logic for this endeavor lives in [Dockerfile](./Dockerfile) and [main.py](./main.py).  
+
+Given an S3 path, use bgzip to compress any *.json files that live within that path while also creating .gzi index files so that data
+can be read out of the compressed files without fully decompressing them.
+
+## Infrastructure and deployment
+We use CloudFormation in [bgz.yml](bgz.yml) to define what we need from AWS batch,
+Elastic Container Registry, and AWS permissions.  To update infrastructure/config for AWS,
+edit the CloudFormation template and then go to [AWS CloudFormation](https://us-east-1.console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks?filteringText=bgzip-batch&filteringStatus=active&viewNested=true) 
```

Comment:
This is pretty cool. I'll have to look it up and read up on it. This seems rather close to Terraform, except I would guess Terraform has a bunch of sugar to make it not as verbose.

Reply:
I'm pretty sure Terraform uses CloudFormation when cloud == AWS.  I think the main advantage for TF is it keeps you from locking in to one cloud, but the disadvantage is you only get a feature set that's common to all clouds.